packages/@aws-cdk/aws-codepipeline/README.md

```diff
@@ -16,6 +16,13 @@ const sourceStage = new Stage(this, 'Source', {
 });
 ```
 
+There's also a utility method on the `Pipeline` class that can be used for this purpose:
```

Comment:
I terms of the README narrative, I would recommend starting with this as the recommended way to add a stage to the pipeline and then mention that it is also possible to instantiate `Stage` directly (I don't even think it's worth providing an example unless there's a clear use case to demonstrate, such as defining a reusable subclass of `Stage`.

Reply:
Done.